pkg/ingestor/parser/cdx_vex/parser_cdx_vex.go

```diff
@@ -0,0 +1,207 @@
+//
+// Copyright 2022 The GUAC Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cdx_vex
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"time"
+
+	cdx "github.com/CycloneDX/cyclonedx-go"
+	"github.com/guacsec/guac/pkg/assembler"
+	"github.com/guacsec/guac/pkg/assembler/clients/generated"
+	"github.com/guacsec/guac/pkg/assembler/helpers"
+	"github.com/guacsec/guac/pkg/handler/processor"
+	"github.com/guacsec/guac/pkg/ingestor/parser/common"
+	"github.com/guacsec/guac/pkg/logging"
+	"strings"
+)
+
+var vexStatusMap = map[cdx.ImpactAnalysisState]generated.VexStatus{
+	"resolved":     generated.VexStatusFixed,
+	"exploitable":  generated.VexStatusAffected,
+	"in_triage":    generated.VexStatusUnderInvestigation,
+	"not_affected": generated.VexStatusNotAffected,
+}
+
+var justificationsMap = map[cdx.ImpactAnalysisJustification]generated.VexJustification{
```

Comment:
@stevemenezes we are missing justifications here? https://cyclonedx.org/docs/1.5/json/#vulnerabilities_items_analysis_justification

@lumjjb and @mlieberman85 same thing here as above status enums

Reply:
Yeah we need to add required missing justifications and statuses. Waiting for confirmation on whether we can combine those with csaf enums or create separate ones.

Reply:
map to existing like above comment